docs/nidcpower/rep_caps.rst

```diff
@@ -35,4 +35,16 @@ channels
         passes a string of :python:`'0, 1, 2'` to the set attribute function.
 
 
+instruments
+-----------
+
+    .. py:attribute:: nidcpower.Session.instruments[]
+
+        .. code:: python
+
+            session.instruments['0-2'].channel_enabled = True
```

Comment:
I think we need to revisit this:

1. '0-2' is not a goo example for `instruments` rep cap
2. `channel_enabled` makes no sense to be set on `instruments`
3. "passes a string of :python:`'0, 1, 2'` to the set attribute function" is irrelevant to Python programmers.

I do believe it should be done separately. Once this is merged please open an issue about this documentation problem.

Reply:
#1602